components/brave_ads/browser/ads_service.h

```diff
@@ -41,6 +41,7 @@ class AdsService : public KeyedService {
   virtual void SetConfirmationsIsReady(const bool is_ready) = 0;
   virtual void IsSupportedRegion(
       IsSupportedRegionCallback callback) = 0;
+  virtual void DisableAds() = 0;
```

Comment:
there is already a method to enable/disable ads, why are we adding a new one?

Reply:
We should be disabling Ads from `bat-native-ads` lib

Reply:
I had created this as I missed the PrefChanged event on `enable_ads(bool)`. Will fix.